lib/liquid_voting/release.ex

```diff
@@ -11,6 +13,26 @@ defmodule LiquidVoting.Release do
     {:ok, _, _} = Ecto.Migrator.with_repo(repo, &Ecto.Migrator.run(&1, :down, to: version))
   end
 
+  @doc """
+  Deletes votes, participants and delegations for the smoke tests organization_id.
+  For use before and after running smoke tests.
+  """
+  def teardown() do
+    organization_id = "bc7eeccb-5e10-4004-8bfb-7fc68536bbd7"
+
+    votes = Voting.list_votes(organization_id)
+    Enum.each(votes, fn vote -> Voting.delete_vote!(vote) end)
```

Comment:
Maybe 

```
organization_id
|> Voting.list_votes
|> Enum.each(fn vote -> Voting.delete_vote!(vote) end)
```

@niklaslong what do you think?

Reply:
and if so, same for participants and delegations

Reply:
function renamed, and organization_id set as module attribute

Reply:
refactored to use pipe construction